awslimits/support.py

```diff
@@ -3,12 +3,14 @@
 from boto3.dynamodb.conditions import Key, Attr
 from collections import namedtuple
 import dateutil.parser
+import os
 
 from dynamo_helpers import create_or_get_table
 
 TICKETS_TABLE_NAME = 'awslimits_tickets'
 LIMITS_TABLE_NAME = 'awslimits_limits'
 NAME_SEPARATOR = " :: "
+LIMIT_ALERT_PERCENTAGE = os.environ.get('LIMIT_ALERT_PERCENTAGE')
```

Comment:
Do you have a specific cutoff you'd recommend for the default? I was using 90% in my local testing